lib/service/Service.ts

```diff
@@ -144,26 +144,35 @@ class Service extends EventEmitter {
   }
 
   /**
-   * Get a list of standing peer orders from the order book for a specified trading pair, or for all
-   * trading pairs if no pair is specified.
+   * Get a list of standing peer orders from the order book for a specified trading pair
    */
   public getOrders = ({ pairId, maxResults }: { pairId?: string, maxResults?: number }) => {
```

Comment:
I think here actually `pairId` and `maxResults` will never be undefined, only the default protobuf values of empty string and 0, so we can drop the `?` nullable operators from the method signature.